### PR TITLE
Ssingulu Phase 1

### PR DESCRIPTION
**Pull request by ssingulu**

### CODE REVIEW

**Comment by Ravsehajsinghpuri** [comment]
@ssingulu 1) Can you add the original tasks in this PR. 2) Can you rename the task variants as taskxxx.1_yyyy, taskxxx.2_yyyy, taskxxx.3_yyyy and so on. For e.g. task547.1_spl_translation_entk_en.json, task547.2_spl_translation_entk_en.json, task547.3_spl_translation_entk_en.json and so on. Do this for all the tasks in this PR
3) Can you also resolve the conflicts

**Comment by Ravsehajsinghpuri** [comment]
@ssingulu Add only the tasks which have been assigned to you. i.e. tasks 475-521 only.

**Comment by ssingulu** [comment]
@Ravsehajsinghpuri Now, I have only included my tasks. i.e. tasks 475-521.

File: tasks/README.md [hunk 1]
```diff
@@ -1425,8 +1509,6 @@ Name | Summary | Category
 `task874_newsroom_title_generation` | Generating title (based on newsroom 1.6m) | Title Generation 
 `task875_newsroom_classification` | Classification of summay (based on newsroom 1.6m) | Classification
 `task876_LC_quad_question_classification` | Generating question from verbalised statement(based on LC-QuAD 2.0) | Question Generation
-`task1001_squad1.1_question_generation` | Generating guestions (based on SQuAD 1.1) | Question Generation
-`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
```
Comment (Ravsehajsinghpuri) [inline tasks/README.md:1429]: add this task as well, it was part of the initial repo

**Comment by ssingulu** [comment]
image

Hey, I have included those files as you can observe in the above image.

**Pull request merged**